plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -395,7 +415,7 @@ object StmtConversionVisitor : FirVisitor<ExpEmbedding, StmtConversionContext<Re
             }
             addStatement(
                 Stmt.If(
-                    EqCmp(lhs, (lhs.type as NullableTypeEmbedding).nullVal).toViper(),
+                    EqCmp(lhs, (lhs.type as NullableTypeEmbedding).nullVal, lhs.pos).toViper(),
```

Comment:
Why does the equality comparison get the position of the left-hand side?  (Same for some of the below.)

Reply:
A typo! 😅